Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h

```diff
@@ -66,6 +66,7 @@ class CLuaElementDefs : public CLuaDefs
     LUA_DECLARE(IsElementDoubleSided);
     LUA_DECLARE(GetElementCollisionsEnabled);
     LUA_DECLARE(IsElementFrozen);
+    LUA_DECLARE(IsElementOnGround);
```

Comment:
You'll have to forward declare the function instead: `bool IsElementOnGround(CEntity* entity);`